src/test/java/org/opensearch/geospatial/GeospatialObjectBuilder.java

```diff
@@ -21,6 +25,9 @@
 
```

Comment:
Is this a utility class for building data for test cases?

Reply:
Yes, thats correct.